### DISCUSSION TITLE
SSL Certificate is invalid and website is getting redirected to cgi-sys/defaultwebpage.cgi

### DISCUSSION THREAD

**Discussion by HarshChaurasia** [vercel/community#6036] [Help] [answered]
### Summary

SSL Certificate of my domain is invalid generated by vercel
and website is getting redirected to cgi-sys/defaultwebpage.cgi
whereas for [save-insta-video-git-main-harshchaurasia.vercel.app](https://save-insta-video-git-main-harshchaurasia.vercel.app/) it is working fine.

### Example

www.saveinstavideo.net

### Steps to Reproduce

Visit www.saveinstavideo.net

**Comment by swarnava** [comment, marked as answer]
Hi, I couldn't reproduce this. Could you send a screencast?

**Comment by HarshChaurasia** [comment]
![image](https://github.com/vercel/community/assets/26917158/e44997e4-0386-4f92-9850-1553dfd47a53)

**Comment by HarshChaurasia** [comment]
it might be because of cache, i am able to open in other device.
Thanks
